src/components/drawing/index.js

```diff
@@ -633,23 +633,10 @@ drawing.bBox = function(node) {
     var testNode = node.cloneNode(true);
     tester.appendChild(testNode);
 
-    // standardize its position - and that of any newlines if we're getting
-    // the bBox of a <text> element directly (not a larger container)
-    // newline <tspan>s need to get `x` and `y` set to match whenever the
-    // containing <text> element gets repositioned, and the line offset
-    // is handled by the attribute `dy` (that we assume is already set)
-    var test3 = d3.select(testNode).attr({
-        x: 0,
-        y: 0,
-        transform: ''
-    });
-
-    if(testNode.nodeName === 'text') {
-        test3.selectAll('tspan.line').attr({
-            x: 0,
-            y: 0
-        });
-    }
+    // standardize its position (and newline tspans if any)
+    d3.select(testNode)
+        .attr('transform', '')
```

Comment:
yes, that's better -> https://github.com/plotly/plotly.js/pull/1792/commits/82ba7eebb16f21933dbae90057984fc5b0a82f09